test/multidim_tests/multidim.jl

```diff
@@ -37,7 +37,16 @@ let X, A  # avoid problems with global variables
 
     @test A ⊆ B
     @test A.v ⊆ B
+    @test A ⊂ B.v
+    @test A ⊂ B
+    @test A.v ⊂ B
     @test A ⊆ B.v
+    @test B ⊇ A
+    @test B.v ⊇ A
+    @test B ⊇ A.v
+    @test B ⊃ A
+    @test B.v ⊃ A
+    @test B ⊃ A.v
```

Comment:
Adding negative tests like

```jl
@test !(B ⊂ A)
```

would be good since currently the function 

```jl
⊂(A, B) = true
```
would pass the tests.

Also there may be corner cases worth testing, like when both `IntervalBox` are the same.

Reply:
Thanks for the suggestions! I have implemented new tests as suggested, including cases where both the function acts on the same `IntervalBox`.